Changelog.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
```

Comment:
Could we keep project structure simpler? It seems like with good commit messages and notes on GitHub releases, this file is just another thing to keep updated and consistent.

See #112 on the need to start doing proper releases.

If we *do* want changelogs in repo, I'd like to see tools such as `beman-tidy` and GitHub PR templates somehow encouraging and requiring them.

Reply:
I'm OK to change the Beman Standard anytime - https://github.com/bemanproject/beman/blob/main/docs/BEMAN_STANDARD.md#changelogmd.

We won't add too often entries in this file. As discussed in previous Beman meeting, the only required line is `CHANGELOG.LIBRARY_STATUS`. The format was chosen to be something universal.

TLDR: Probably we'll document only library status changed + releases (such - this release has Pxxx and Pyyy implementations). All concerns related to "how big it will be this file" were already address.

@bretbrownjr , is that OK for you?